docs/book/src/topics/full-multitenancy-implementation.md

```diff
@@ -0,0 +1,259 @@
+            - system:serviceaccount:capa-system:capa-controller-manager
+            - system:serviceaccount:capa-eks-control-plane-system:capa-eks-control-plane-controller-manager # Include if also using EKS
+EOL
+```
+
+Let's provision the Manager account with:
+
+```
+clusterawsadm bootstrap iam create-cloudformation-stack --config bootstrap-manager-account.yaml
+```
+
+## Manager cluster
+
+The following commands assume you have the AWS credentials for the Manager account exposed, and your kube context is pointing to the bootstrap cluster.
+
+### Install cluster API provider in the bootstrap cluster
+
+```bash
+export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile)
+export EKS=true
+export EXP_MACHINE_POOL=true
+clusterctl init --infrastructure aws --target-namespace capi-providers
+```
+
+### Generate the cluster configuration
+
+**NOTE:** You might want to update the Kubernetes and VPC addon versions to one of the available versions when running this command.
+
+- [Kubernetes versions](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
+- [VPC CNI add-on versions](https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html) don't forget to add the `v` prefix
+
+```bash
+export AWS_SSH_KEY_NAME=default
+export VPC_ADDON_VERSION="v1.10.2-eksbuild.1"
+clusterctl generate cluster manager --flavor eks-managedmachinepool-vpccni --kubernetes-version v1.20.2 --worker-machine-count=3 > manager-cluster.yaml
+```
+
+### Apply the cluster configuration
+
+```bash
+kubectl apply -f manager-cluster.yaml
+```
+
+**WAIT**: time to have a drink, the cluster is creating and we will have to wait for it to be there before continuing.
+
+### IAM OIDC Identity provider
+
+Follow AWS documentation to create an OIDC provider https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html
+
+### Update the TrustStatement above
+
+```bash
+export OIDC_PROVIDER_ID=<OIDC_ID_OF_THE_CLUSTER>
+```
+
+run the [Prepare the manager account](./full-multitenancy-implementation.md#prepare-the-manager-aws-account-0-account) step again
+
+### Get manager cluster credentials
+
+```bash
+kubectl --namespace=default get secret manager-user-kubeconfig \
+   -o jsonpath={.data.value} | base64 --decode \
+   > manager.kubeconfig
+```
+
+### Install the CAPA provider in the manager cluster
+
+Here we install the Cluster API providers into the manager cluster and create a service account to use the `controllers.cluster-api-provider-aws.sigs.k8s.io` role for the Management Components.
+
+```bash
+export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile)
+export EKS=true
+export EXP_MACHINE_POOL=true
+export AWS_CONTROLLER_IAM_ROLE=arn:aws:iam::${AWS_MANAGER_ACCOUNT_ID}:role/controllers.cluster-api-provider-aws.sigs.k8s.io
+clusterctl init --kubeconfig manager.kubeconfig --infrastructure aws --target-namespace capi-providers
```

Comment:
@richardcase forgot if there was an issue with self-managed clusters for EKS. Would `clusterctl move` work for creating an EKS cluster then pivoting there?